### PR TITLE
Add more on config troubleshooting for desktop wallet

### PR DESCRIPTION
**Pull request by candida-d**

### CODE REVIEW

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { NetworkConfigAddress, NetworkConfigAddressText } from '@site/src/components/NetworkConfigAddress';
+import CodeBlock from '@theme/CodeBlock';
+
+
+If you have issues with the Vega Wallet desktop app, you may find the solution below. If not, see the [reporting bugs](#reporting-bugs) section below. 
+
+## Not yet supported
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:14]: what do you think ?
```suggestion
## Missing features
```

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Customise key details
+* Isolate keys
+* Build and send transactions 
+
+## 'Wallet already exists' error when recovering wallet
+### Problem
+You recover a wallet using your recovery phrase. When you enter the wallet name, you get an error of `Wallet with this name already exists`. 
+
+### Solution
+The wallet you are trying to recover already exists on your device. You can use your existing wallet with the Vega Wallet desktop app. On the onboarding screen, choose `Use existing` to connect to an existing Vega wallet.
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:25]:
```suggestion
A wallet with the same name already exist on your device. You can change the name of the wallet you are trying to restore, or you can use your existing wallet with the Vega Wallet desktop app. On the onboarding screen, choose `Use existing` to connect to an existing Vega wallet. 
```

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+The wallet you are trying to recover already exists on your device. You can use your existing wallet with the Vega Wallet desktop app. On the onboarding screen, choose `Use existing` to connect to an existing Vega wallet. 
+
+If you see the error message `Wallet with this name already exists` but you do not see that wallet on the Wallets screen, then report the issue on the **[feedback board](https://github.com/vegaprotocol/feedback/discussions/)**. 
+
+## Wallet recovery provides wrong keys 
+### Problem
+You recover a wallet using your recovery phrase, and the public key you see doesn't match the key you expect (such as if you expected that key to have assets associated with it, and it doesn't). 
+
+### Possible solutions
+* Choose the alternate wallet version number alongside your recovery phrase. If you recovered and chose "version 2", then you should try using the same recovery phrase but choose "version 1".
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:34]:
```suggestion
* Choose the alternate key derivation version number alongside your recovery phrase. If you recovered and chose "version 2", then you should try using the same recovery phrase but choose "version 1".
```

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* Ensure that your recovery phrase includes the exact same words, in the exact same order, as they were provided to you when you first created your wallet.
+
+## Wallet will not connect
+If you have trouble connecting, especially after a software release to the network you're using, you may need to delete your network configuration and use the default configuration. This is particularly likely after the 0.71.4 release.
+
+### Possible solution
+* Navigate to `Manage network`. Remove the network. Restart the app. Once you remove the network and restart the app, the app will download the latest version of the config for you.
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:41]:
```suggestion
1. Navigate to `Manage network`.
2. Remove the network.
3. Restart the app.

Once you remove the network and restart the app, the app will download the latest version of the config for you.
```

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## No networks available
+If you chose not to import the available networks when setting up the wallet app, you will need to import them manually. Use the .toml URLs below to import a network.
+
+* **Mainnet** network (run by validators): <NetworkConfigAddress frontMatter={frontMatter} label="mainnet1.toml" network="mainnet"/>
+* **Fairground** network: <NetworkConfigAddress frontMatter={frontMatter} label="fairground.toml" network="fairground"/>
+* **Validator testnet** network: [testnet2.toml ↗](https://raw.githubusercontent.com/vegaprotocol/networks/master/testnet2/testnet2.toml)
+
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:49]: This is no longer _completely_ true.

When starting up the application, the default networks configuration will be imported, no matter what. The check is done on every startup.

So, I think we can remove this section.

**Review by ValentinTrinque — CHANGES_REQUESTED** [review]

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md (hunk 4 above)
Comment (candida-d) [reply docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:41]: fair enough 😆

File: docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md (hunk 5 above)
Comment (candida-d) [reply docs/tools/vega-wallet/desktop-app/latest/troubleshooting.md:49]: Ok! Will remove.

**Review by ValentinTrinque — APPROVED** [review]

**Pull request merged**